transmart-core-db-tests/test/integration/org/transmartproject/db/ontology/OntologyTermTagsResourceServiceTests.groovy

```diff
@@ -118,13 +118,25 @@ class OntologyTermTagsResourceServiceTests {
                     hasProperty('key', equalTo(key)),
```

Comment:
@gijskant IMHO that's not stopper for this PR to be merged. As soon as it works.